### PR TITLE
Install 1.x instructions

### PR DESCRIPTION
**Pull request by namelessperson0**
Fixes https://github.com/EmmanuelRoux/ngx-matomo/issues/50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,17 +43,17 @@ Matomo (fka. Piwik) client for Angular applications. Compatible with Matomo 3 or
 
 | NgxMatomo | Angular     | Matomo        |
 | --------- | ----------- | ------------- |
-| latest    | 9.x to 12.x | Matomo 3 or 4 |
+| 1.x    | 9.x to 12.x | Matomo 3 or 4 |
 
 ## Installation
 
-`npm install --save @ngx-matomo/tracker`
+`npm install --save @ngx-matomo/tracker@^1
```
Comment (EmmanuelRoux) [inline README.md]: Missing ending backquote

File: README.md [hunk 2]
```diff
@@ -43,17 +43,17 @@ Matomo (fka. Piwik) client for Angular applications. Compatible with Matomo 3 or
...
 
 Optionally, if you use [Angular router](https://angular.io/guide/router) and want to easily track routed components, you
 may also install the NgxMatomo router adapter
 ([see documentation below](#tracking-page-view-with-angular-router--ngxmatomoroutermodule)):
 
-`npm install --save @ngx-matomo/router`
+`npm install --save @ngx-matomo/router@^1
```
Comment (EmmanuelRoux) [inline README.md]: Missing ending backquote

**Comment by EmmanuelRoux** [comment]
Hi @namelessperson0 thank you for this PR

Can you please fix the missing backquotes (see comments) and verify there is no other formatting issue?

**Review by EmmanuelRoux — CHANGES_REQUESTED** [review]

**Comment by EmmanuelRoux** [comment]
@namelessperson0 do you plan to push the changes so I can merge your PR?

**Comment by namelessperson0** [comment]
> @namelessperson0 do you plan to push the changes so I can merge your PR?

The backtick issue has been fixed now :)

**Comment by EmmanuelRoux** [comment]
@namelessperson0 I've pushed a commit to fix formatting issues, now merged. Thanks for the PR! Fell free to contribute again :)

**Pull request merged**